src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -60,8 +60,27 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
         Map<Id,Account> mapAccountIdAccount = null;
         boolean acctHasAddress = false;
         boolean acctChanged = false;
+        boolean isBefore = (triggerAction == TDTM_Runnable.Action.BeforeUpdate || triggerAction == TDTM_Runnable.Action.BeforeInsert);
```

Comment:
@force2b Added logic similar to ADDR_Account_TDTM, checking for changed address in BeforeUpdate.